_posts/2021-08-10-memn.md

```diff
@@ -0,0 +1,150 @@
+
+## Einleitung
+
+In den letzten Jahren haben die Lerninhalte über die Robotik an Schulen, hier Oberstufe, als interdisziplinäre, projektbasierte Lernaktivität zugenommen. Sie stützen sich stark auf den Informatikunterricht und fördern ein problem - basiertes Denken. Das Ziel des Einsatzes von Robotern im Unterricht ist es, den Schülern die Möglichkeit zu geben, das Verhalten eines physischen Modells über eine virtuelle Umgebung zu steuern und über das problem – basierte Denken hinaus in das «Computational Thinking» zu leiten. In diesem Erlebnisbericht engl. Report of experience wird der Begriff «Computational Thinking» eingeführt und an den Schülern der Sekundarschule 1. – 3. der schweizerischen Schulen hergeleitet. (https://roboesl.eu/wp-content/uploads/2017/08/Robotics-in-Education-Education-in-Robotics.pdf)
+
+Eine erfolgreiche Einführung der Robotiktechnologie ist nicht nur die Frage des Zugangs an neuen Geräten, sondern das Beibringen und die Ausführung der computerbasierten Denkweise. Der Roboter ist lediglich ein weiteres Werkzeug, um neue technische Ideen zu generieren und vielleicht sogar Lösungen zu kreieren. Eine Zusammenarbeit mit den Lehrern der Oberstufenschule Wädenswil in Kanton Zürich (Schweiz), eine zukunftsorientierte Bildungsphilosophie, ein gut konzipierter Kursplan und eine unterstützende Lernumgebung sind unsere Basiselemente für das erfolgreiche Einsetzen und Einführen des Projekts «Machen Können» an den Schulen, bei dem die technischen Tools der aus China im Jahre 2008 gegründeten Firma Seeed Technology Co., Ltd. verwendet werden. (https://www.seeedstudio.com/blog/2017/11/09/when-grove-zero-meets-microbit/) 
+
+Im Jahr 2020(?) lancierte die Hasler Stiftung ein Bildungsprojekt, welches als Projektziel die “Förderung eines nachhaltigen Informatikunterrichts in der Oberstufe” hat und unterstützt die Zürcher Hochschule für angewandte Wissenschaften (ZHAW) in deren Forschung in die Digitalisierung. Das daraus entstandene Projekt «Machen Können» stellt selbstlernende Kurse da, welche an allen Klassenstufen selbstständig umsetzbar sind. "Machen Können" ermöglicht den Schülern der Oberstufe einen sicheren Umgang mit Mikrocontrollern, Sensoren und Aktuatoren im Informatikunterricht und die SchülerInnen erlernen einen nachhaltigen Lernerlebnis. Dadurch wollen wir ihre Fähigkeiten fördern, sich in einer digitalen Gesellschaft mit immer mehr eingebetter Informationstechnologie besser zurechtzufinden und diese kreativ mitzugestalten. Das StarterKit der Firma Seeed Technology Co., Ltd. bietet eine Möglichkeit dazu. (D. Bajka, 2021) Siehe dazu mehr im Blogeintrag [Physical Computing in der Schule](https://dxi.ai/2021/03/22/bajk/) 
```

Comment:
Uff. Das ist nicht ganz korrekt. 

“Förderung eines nachhaltigen Informatikunterrichts in der Oberstufe” ist ein Förderprogramm. 

Damit werden Informatik-zentrische wissenstransfer Projekte von Hochschulen an Oberstufen gefördert. 

Machen können wird aus diesem Topf gefördert. aber das würde ich nicht so hervorheben. 

Kurz: 

```
Machen Können wird im Rahmen der  “Förderung eines 
nachhaltigen Informatikunterrichts in der Oberstufe” 
durch die Hasler Stiftung gefördert. 
```

Das reicht vollkommen. 

Diesen Satz würde ich ganz ans Ende des Beitrags angeben.